README.md

```diff
@@ -1 +1,47 @@
-# bons.ai
\ No newline at end of file
+# bons.ai
+
+bons.ai is web application, which allows you to perform Question Answering task on pdf files uploaded or text provided. Currently we have developed a simple python script to use hugging face models to perform question answering task.
+
+## Setup
+
+### Setting up Environment
+
+1. Create python virtual environment using this command
+
+```
+python -m venv virtualenv
+```
+
+2. Install all the required modules.
```

Comment:
You need to activate the virtual environment before you can install the requirements. If you install the requirements before you start you env then the installations will take place in your global python installation which is definitely not what you want to do.

Reply:
I will add a step to activate virtual environment while installing requirements.